estate/models/estate_property.py

```diff
@@ -0,0 +1,23 @@
+from dateutil.relativedelta import relativedelta
+from odoo import fields, models
+
+
+class Property(models.Model):
+    _name = "estate_property"
```

Comment:
You should use dots instead of underscores for the model names. (Sorry for the bad info previously).
```suggestion
    _name = "estate.property"
```